ts/Extensions/Annotations/Types/Measure.ts

```diff
@@ -275,32 +271,33 @@ function min(
             this.yAxisMax
         );
 
-    let min: (''|number) = Infinity,
+    let min: number = Infinity,
         isCalculated = false; // To avoid Infinity in formatter
 
-    series.forEach((serie): void => {
+    series.forEach((s): void => {
         if (
-            serie.visible &&
-            serie.options.id !== 'highcharts-navigator-series'
+            s.visible &&
+            s.options.id !== 'highcharts-navigator-series'
         ) {
-            serie.points.forEach((point: Point): void => {
+            s.points.forEach((point: Point): void => {
                 if (
                     !point.isNull &&
-                    (point.y as any) < min &&
-                    (point.x as any) > ext.xAxisMin &&
-                    (point.x as any) <= ext.xAxisMax &&
-                    (point.y as any) > ext.yAxisMin &&
-                    (point.y as any) <= ext.yAxisMax
+                    point.y &&
```

Comment:
What if.. 
Is this something we could refactor? Instead of using the same code three times?